frontend/apps/reader/modules/readermenu.lua

```diff
@@ -305,6 +328,48 @@ dbg:guard(ReaderMenu, 'setUpdateItemTable',
         end
     end)
 
+function ReaderMenu:saveDocumentSettingsAsDefault()
+    if self.ui.rolling then
```

Comment:
Do we have a way to pull those from the doc's configurable instance automatically?

(This would mean less maintenance burden in case they ever change).

Reply:
I'll think of this

Reply:
This big listing of saveSetting does not really have place in a ReaderMenu.
I would put it in a new module that you would just use here - with resetDocumentSettings() which also has quite nothing to do with FilemanagerUtil :)
`require("apps/filemanager/filemanagerutil").resetDocumentSettings(current_file)`

Reply:
> This big listing of saveSetting does not really have place in a ReaderMenu.

It is rather short now, maybe keep it here?

Reply:
As you prefer.
Still, resetDocumentSettings() which is only used in ReaderMenu may probably be moved where the new saveDocumentSettingsAsDefault() is.

Reply:
` resetDocumentSettings()` is near `purgeSettings()`, they have similar structures and goals.
I agree that ReaderMenu should be just a frame but cannot find better place for these "doc settings" functions.